modules/xz/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://tukaani.org/",
+    "maintainers": [
+        {
+          "email": "dev@rdxip.com",
+          "github": "Attempt3035",
+          "name": "Luke Aguilar"
+        }
```

Comment:
I am also willing to maintain this. @Attempt3035 can you add me?
```
        {
            "email": "julian.amann@tum.de",
            "github": "Vertexwahn",
            "name": "Julian Amann"
        }
```

Reply:
Awesome! Thanks! Added👍